host/xtest/pkcs11_1000.c

```diff
@@ -1988,12 +1988,17 @@ static CK_ATTRIBUTE cktest_generate_gensecret_object_valid1[] = {
 	{ CKA_SIGN, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
 	{ CKA_VERIFY, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
 	{ CKA_VALUE_LEN, &(CK_ULONG){16}, sizeof(CK_ULONG) },
+	{ CKA_VALUE_LEN, &(CK_ULONG){16}, sizeof(CK_ULONG) },
 };
 
 /* Valid template to generate a generic secret with only VALUE_LEN */
 static CK_ATTRIBUTE cktest_generate_gensecret_object_valid2[] = {
 	{ CKA_VALUE_LEN, &(CK_ULONG){16}, sizeof(CK_ULONG) },
 	{ CKA_SIGN, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
+	{ CKA_SENSITIVE, &(CK_BBOOL){CK_FALSE}, sizeof(CK_BBOOL) },
+	{ CKA_EXTRACTABLE, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
+	/* Keep derive attribute at end of template */
```

Comment:
i think this is a bit fragile in the long term. I would prefer there are 2 specific objects when a single one does not match the expected content. What do you think?

Reply:
Ok. I will add another template. Just wanted to avoid duplicating too many templates so thought of adding this at end , and then not using the last attribute there.